scapy/main.py

```diff
@@ -182,25 +182,29 @@ def list_contrib(name=None):
 
 
 def save_session(fname=None, session=None, pickleProto=-1):
```

Comment:
Could you add a docstring ?

For example, """save current Scapy session to the file specified in the fname arg"""

Reply:
Done